sda-commons-server-dropwizard/docs/metadata-context.md

```diff
@@ -0,0 +1,189 @@
+# The Metadata Context
+
+
+## Purpose
+
+The metadata context keeps contextual information of a business process across all involved
+microservices.
+The information that is stored in the metadata context is solely defined by the environment.
+An environment may not define any metadata.
```

Comment:
This sounds confusing, doesn’t the second sentence contradict the first?

Reply:
Hmmm … felt more clear a few minutes ago.
The second sentence should clarify that it is not needed to define what's include in the metadata context. An environment may or may not define fields for the context. But it is only the environment/customer/operations that has the right to define it. No service can require that such configuration exists.
I will think about that but I'm also happy to get a suggestion.

Reply:
I think "may not" would be translated to "darf nicht". After reading your explanation we could rephrase to "it's optional" or "it's not mandatory".